docs/source/technical_tutorials/access/accessing_data.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
```

Comment:
<p>My only issue with this cell is I have to remove the [0] - with it, I get an error, but without it I get the OnlineAccessURL.</p><p> </p><p>edit: with the [0] it gives me an error in the old ADE, but not the new one, so this should be fine</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>